website/docs/reference/supported_commands.md

```diff
@@ -7,7 +7,7 @@ sidebar_position: 1
 ## Query commands
 
 | Command         | Argument                   | Status | Comments                                                   |
-| --------------- | -------------------------- | ------ | ---------------------------------------------------------- |
+| --------------- | -------------------------- | ------ |------------------------------------------------------------|
```

Comment:
Note that other columns and all other tables have a padding space before and after `|`

Reply:
Fixed.